spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProduceItemsHelper, type: :helper do
+  context 'when viewing the produce items index' do
+    let(:produce_item) { ProduceItem.create(name: 'carambola', category: 'fruit') }
+    let(:item_season_jan) { ItemSeason.create(produce_item_id: produce_item.id, month_index: 1, country_code: 'br') }
```

Comment:
**issue:** country codes should always be uppercase as per the [ISO 3166-1 definition](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)